src/scripts/components/widgets/FilesUploader.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import styled, { css } from 'styled-components'
 import FilesUploaderSvg from '../foundations/svgs/FilesUploaderSvg'
 import Button from '../foundations/buttons/Button'
-import TextField from '../foundations/forms/TextField'
+import TextField from '../widgets/forms/TextField'
```

Comment:
Not a big deal but this file is already in `widgets` so this can be `./forms` or just `components/widgets/forms/etc`